packages/api/package.json

```diff
@@ -33,18 +34,25 @@
     "decompress-tarbz2": "^4.1.1",
     "decompress-tarxz": "^3.0.0",
     "express": "^4.17.1",
+    "express-jwt": "^5.3.1",
     "express-pino-logger": "^4.0.0",
+    "express-session": "^1.17.0",
+    "jsonwebtoken": "^8.5.1",
+    "keycloak-connect": "^9.0.0",
```

Comment:
since we are using `express-jwt`. 

`express-session` and `keycloak-connect` should be removed

Reply:
:+1: thanks, good catch.